test/extended/prometheus/prometheus.go

```diff
@@ -176,6 +176,29 @@ var _ = g.Describe("[Feature:Prometheus][Conformance] Prometheus", func() {
 
 			e2e.Logf("Watchdog alert is firing")
 		})
+		g.It("should have important platform topology metrics", func() {
+			oc.SetupProject()
+			ns := oc.Namespace()
+			execPod := exutil.CreateCentosExecPodOrFail(oc.AdminKubeClient(), ns, "execpod", nil)
+			defer func() { oc.AdminKubeClient().CoreV1().Pods(ns).Delete(execPod.Name, metav1.NewDeleteOptions(1)) }()
+
+			tests := map[string]bool{
+				// track infrastructure type
+				`count(cluster_infrastructure_provider{type!=""}) == 1`: true,
```

Comment:
This test syntax is not yet supported on `release-4.2` and earlier branches. Support for it is on my todo list.

Reply:
That's fine, I'm going to wait until 4.3 passes and then tweak these.  I don't even expect these to pass really because I think we changed some of the counts.